modules/mobile/spec/request/user_request_spec.rb

```diff
@@ -290,6 +293,49 @@
           )
         end
       end
+
+      context 'with a user who does not have access to schedule appointments' do
+        let(:user_request) do
+          iam_sign_in(FactoryBot.build(:iam_user, :no_participant_id))
```

Comment:
Can you explain to me why this user does not have access to schedule appointments? The requirements for this policy are that the user have Level of Assurance 3 and have access to VA facilities. I don't really understand how levels of assurance work. Why does `no_participant_id` result in low LOA? 

Also, are low LOA and lack of facilities tied together? In other words, is it possible to meet one condition but not the other? The way we're testing here, we seem to assume that low LOA and lack of facilities go hand-in-hand. Is that the case? Can someone have LOA 3 but lack facilities, or visa versa?

Reply:
`no_participant_id ` caused no facilities to be stored, so I utilized it to test the condition that user must have registered facilities get pass the policy. There may be other reasons no participant id could cause this though so I updated the spec to create a user that explicitly had the facilities taken out. Also created another spec with a loa2 user to test the other condition, though this shouldn't come up in production since all mobile users are loa3.